libledger/LedgerManager.h

```diff
@@ -72,21 +64,14 @@ class LedgerManager
                                      << std::to_string(_groupId);
             return false;
         }
-        std::shared_ptr<LedgerInterface> ledger =
-            std::make_shared<T>(m_service, _groupId, m_keyPair, _baseDir, configFileName);
-        LedgerManager_LOG(INFO) << "[initSingleLedger] [GroupId]:  " << std::to_string(_groupId);
-        ledger->setChannelRPCServer(m_channelRPCServer);
-        bool succ = ledger->initLedger();
-        if (!succ)
-            return false;
         m_ledgerMap.insert(std::make_pair(_groupId, ledger));
         {
             WriteGuard l(x_groupListCache);
             m_groupListCache.insert(_groupId);
         }
         return true;
     }
-
+    bool isLedgerExist(dev::GROUP_ID const& _groupId) { return m_ledgerMap.count(_groupId); }

```

Comment:
It's no need to confirm the groupId exists or not since https://github.com/FISCO-BCOS/FISCO-BCOS/pull/701/files#diff-ff085bb116eced68538f1fa79c1bbe78R61 has confirmed.

Reply:
> It's no need to confirm the groupId exists or not since https://github.com/FISCO-BCOS/FISCO-BCOS/pull/701/files#diff-ff085bb116eced68538f1fa79c1bbe78R61 has confirmed.

link is wrong, please tell me where